verifiers/internal/gha/verifier.go

```diff
@@ -131,7 +131,7 @@ func verifyNpmEnvAndCert(env *dsse.Envelope,
 
 	// Users must always provide the builder ID.
 	if builderOpts == nil || builderOpts.ExpectedID == nil {
-		return nil, fmt.Errorf("builder ID is empty")
+		return nil, fmt.Errorf("%w: empty", serrors.ErrorInvalidBuilderID)
```

Comment:
nit: "empty" by itself is a bit vague
```suggestion
		return nil, fmt.Errorf("%w: no expected builder ID", serrors.ErrorInvalidBuilderID)
```